src/openrct2/paint/tile_element/Paint.Surface.cpp

```diff
@@ -741,8 +739,9 @@ static void viewport_surface_draw_tile_side_bottom(
 
         image_id = get_tunnel_image(edgeStyle, tunnelType) + (edge == EDGE_BOTTOMRIGHT ? 2 : 0) + 1;
         PaintAddImageAsParent(
-            session, image_id, offset.x, offset.y, tunnelBounds.x, tunnelBounds.y, boundBoxLength - 1,
-            curHeight * COORDS_Z_PER_TINY_Z, tunnelTopBoundBoxOffset.x, tunnelTopBoundBoxOffset.y, boundBoxOffsetZ);
+            session, image_id, { offset.x, offset.y, curHeight * COORDS_Z_PER_TINY_Z },
+            { tunnelBounds.x, tunnelBounds.y, boundBoxLength - 1 },
+            { tunnelTopBoundBoxOffset.x, tunnelTopBoundBoxOffset.y, boundBoxOffsetZ });
```

Comment:
```suggestion
            { tunnelTopBoundBoxOffset, boundBoxOffsetZ });
```

Reply:
Theres no conversation from LocationXY16 to CoordsXY or CoordsXYZ, this won't work.